### PR TITLE
v24.2.0_cycle24 release changelog

### PR DESCRIPTION
**Pull request by llewellyn-sl**

### CODE REVIEW

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 1]
```diff
@@ -5,13 +5,36 @@ date: "4 July 2024"
...
+- Feature: define cache configuration once, only override changes
+- Improvement: Data Explorer: Highlight invalid custom datalinks
+- Improvement: Disable AWS and Google Batch spot instance auto retry
+- Improvement: Data explorer: Remove folder validation
+- Improvement: Update 204 pipeline schema response description
+- Improvement: Patch workflow revision
+- Improvement: Render Parameters UI correctly with new schemas based on 2020-12 draft
+- Improvement: Upgrade to Angular 16
+- Improvement: Pipeline launch form: Delegate form creation to a form builder service
+- Improvement: Add Posthog integration
```
Comment (swampie) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:22]: @drewdipalma any additional comment on posthog wording before we go live with the docs?

Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:22]:
```suggestion
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 2]
```diff
@@ -5,13 +5,35 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
-### v24.2.0_cycle23 Cloud - 2 September 2024
+### v24.2.0_cycle24 25 September 2024
+
+- Feature: Azure service principal credentials
+- Feature: Data Studios: Add direct Data Explorer browse link to mounted data items
+- Feature: Data Studios: Make StudioDetailsPage tabs routable
+- Feature: define cache configuration once, only override changes
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:13]:
```suggestion
- Feature: Define cache configuration once, only override changes
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 3]
```diff
@@ -5,13 +5,35 @@ date: "4 July 2024"
...
+- Improvement: Data Explorer: Highlight invalid custom datalinks
+- Improvement: Disable AWS and Google Batch spot instance auto retry
+- Improvement: Data explorer: Remove folder validation
+- Improvement: Update 204 pipeline schema response description
+- Improvement: Patch workflow revision
+- Improvement: Render Parameters UI correctly with new schemas based on 2020-12 draft
+- Improvement: Upgrade to Angular 16
+- Improvement: Pipeline launch form: Delegate form creation to a form builder service
+- Fix: Connection to Redis SSL server
+- Fix: pairingId declaration in API schema
```
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:23]:
```suggestion
- Fix: `pairingId` declaration in API schema
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx (hunk 1 above)
Comment (llewellyn-sl) [reply platform_versioned_docs/version-24.1/cloud/changelog.mdx]: Per Drew, removing the Posthog line.

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 4]
```diff
@@ -5,13 +5,35 @@ date: "4 July 2024"
...
-### v24.2.0_cycle23 Cloud - 2 September 2024
+### v24.2.0_cycle24 — 25 September 2024
+
+- Feature: Azure service principal credentials
+- Feature: Data Studios: Add direct Data Explorer browse link to mounted data items
+- Feature: Data Studios: Make StudioDetailsPage tabs routable
+- Feature: Define cache configuration once, only override changes
+- Improvement: Data Explorer: Highlight invalid custom datalinks
+- Improvement: Disable AWS and Google Batch spot instance auto retry
+- Improvement: Data explorer: Remove folder validation
```
Comment (christopher-hakkaart) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:16]:
```suggestion
- Improvement: Data Explorer: Remove folder validation
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 5]
```diff
@@ -5,13 +5,35 @@ date: "4 July 2024"
 tags: [changelog]
 ---
 
-### v24.2.0_cycle23 Cloud - 2 September 2024
+### v24.2.0_cycle24 — 25 September 2024
+
+- Feature: Azure service principal credentials
+- Feature: Data Studios: Add direct Data Explorer browse link to mounted data items
+- Feature: Data Studios: Make StudioDetailsPage tabs routable
```
Comment (christopher-hakkaart) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:12]: One word for StudioDetailsPage?

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx [hunk 6]
```diff
@@ -5,13 +5,35 @@ date: "4 July 2024"
...
+- Improvement: Patch workflow revision
+- Improvement: Render Parameters UI correctly with new schemas based on 2020-12 draft
+- Improvement: Upgrade to Angular 16
+- Improvement: Pipeline launch form: Delegate form creation to a form builder service
+- Fix: Connection to Redis SSL server
+- Fix: `pairingId` declaration in API schema
+- Fix: Copying empty list for cloud data links
+- Fix: Visual glitches in launch form
+- Fix: Datalink status always null
+- Fix: Update lastUsed on compute environment when creating a data studio job
```
Comment (christopher-hakkaart) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:27]: One word lastUsed?

**Review by christopher-hakkaart — APPROVED** [review]
I've added a couple of comments and one suggestion. Comments are to check they aren't typos. Not blockers, so approved so you can merge when you're ready.

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx (hunk 5 above)
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:12]:
```suggestion
- Feature: Data Studios: Make studio details page tabs routable
```

File: platform_versioned_docs/version-24.1/cloud/changelog.mdx (hunk 6 above)
Comment (llewellyn-sl) [inline platform_versioned_docs/version-24.1/cloud/changelog.mdx:27]:
```suggestion
- Fix: Update last used field in compute environment when creating a data studio job
```

**Pull request merged**